client/src/containers/ChooseCampaign.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+class ChooseCampaign extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedOption: null
+    }
+    this.handleOptionChange = this.handleOptionChange.bind(this)
+  }
+
+  handleOptionChange(e) {
+    this.setState({
+      selectedOption: e.target.value
+    });
+  }
+
+  handleFormSubmit(e) {
+    e.preventDefault();
+    const selectedOption = this.state.selectedOption;
+    console.log('You have selected:', selectedOption);
+    if(selectedOption === 'no-match') {
+      this.props.router.push('/new-campaign/address')
+    }
+  }
+
+  renderNearbyCampaigns(nearbyCampaignsArr, selectedOption, handleOptionChange) {
+    return (
+      nearbyCampaignsArr.map(c => (
+        <div className="radio" key={c.street_address}>
+          <label>
+            <input
+              type="radio"
+              value={c.street_address}
+              checked={selectedOption === c.street_address }
+              onChange={(e) => handleOptionChange(e)} />
```

Comment:
I'll try to figure out a way to bind these in the constructor so that we don't have to react new functions every time.